apps/pi-drone-detector/scripts/download_public_datasets.py

```
@@ -0,0 +1,424 @@
+- Morning dew on webs
+- Backlit webs
+
+## Tips
+- These are negative examples (not drones)
+- Capture in same conditions as deployment
+""")
+
+    print(f"  Guides created at: {guides_dir}")
+
+
+def create_combined_dataset(output_dir: Path, classes: List[str]):
```

Ruff (0.14.10)

Unused function argument: `classes`

(ARG001)